saleae/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.11.0'
+__version__ = '0.12.0'
```

Comment:
Can you drop this from the changeset here? I think it's valuable to have the version bumps as isolated, no-code patches that act as a changelog. I'll bump the version when I put out a release.

Reply:
ACK.